app/views/admins/panel/index.html.erb

```diff
@@ -393,10 +393,11 @@
 
       // Pop-up form functions
 
-      function show_bg_shadow() {
+      function show_bg_shadow(form_id) {
```

Comment:
This function is about setting background shadow, this functionality does not belong here.